google/cloud/aiplatform/explain/lit.py

```diff
@@ -148,7 +154,63 @@ def input_spec(self) -> lit_types.Spec:
 
     def output_spec(self) -> lit_types.Spec:
         """Return a spec describing model outputs."""
-        return self._output_types
+        output_spec_dict = dict(self._output_types)
+        if self.attribution_explainer:
+            output_spec_dict["feature_attribution"] = lit_types.FeatureSalience(
+                signed=True
+            )
+        return output_spec_dict
+
+    def _load_model(self, model: str):
+        """Loads a TensorFlow saved model and populates the input and output signature attributes of the class.
+            Args:
+              model: Required. A string reference to a TensorFlow saved model directory.
+            Raises:
+                ValueError if the model has more than one input tensor or more than one output tensor.
+            """
+        self._loaded_model = tf.saved_model.load(model)
+        serving_default = self._loaded_model.signatures[
+            tf.saved_model.DEFAULT_SERVING_SIGNATURE_DEF_KEY
+        ]
+        _, self._kwargs_signature = serving_default.structured_input_signature
+        self._output_signature = serving_default.structured_outputs
+
+        if len(self._kwargs_signature) != 1:
+            raise ValueError("Please use a model with only one input tensor.")
+
+        if len(self._output_signature) != 1:
+            raise ValueError("Please use a model with only one output tensor.")
+
+    def _set_up_attribution_explainer(
+        self, model: str, attribution_method: str = "integrated_gradients"
+    ):
+        """Populates the attribution explainer attribute of the class."""
+        try:
+            import explainable_ai_sdk
+            from explainable_ai_sdk.metadata.tf.v2 import SavedModelMetadataBuilder
+        except ImportError:
+            print(
+                "Skipping explanations because the Explainable AI SDK is not installed."
```

Comment:
Preference for either `logging.info` or `logging.warning`

Reply:
Chose logging.info.